scripts/bump.js

```diff
@@ -0,0 +1,19 @@
+const fs = require("fs");
+const path = require("path");
+const { exec } = require("@actions/exec");
+
+(async () => {
+  await exec("pnpm changeset", ["version"]);
+  const apps = fs.readdirSync("apps");
+  apps.map((app) => {
+    const newVersion = `${require(`../apps/${app}/package.json`).version}`;
+    const dockerFilePath = path.join(`apps/${app}/contrib/dokku`, "Dockerfile");
```

Comment:
```suggestion
    const dockerFilePath = path.join("apps", app, "contrib/dokku/Dockerfile");
```


Why use `path.join` if we're using `${}`? Lets pick one.